sql/ddl_internal.sql

```diff
@@ -20,3 +20,8 @@ AS '@MODULE_PATHNAME@', 'ts_data_node_block_new_chunks' LANGUAGE C VOLATILE;
 -- distributed hypertables
 CREATE OR REPLACE FUNCTION _timescaledb_internal.allow_new_chunks(data_node_name NAME, hypertable REGCLASS = NULL) RETURNS INTEGER
 AS '@MODULE_PATHNAME@', 'ts_data_node_allow_new_chunks' LANGUAGE C VOLATILE;
+
+CREATE OR REPLACE FUNCTION _timescaledb_internal.refresh_continuous_aggregate(
```

Comment:
Since the public version is a procedure, this one should probably also be a procedure. The return value is not useful anyway.
```suggestion
CREATE OR REPLACE PROCEDURE _timescaledb_internal.refresh_continuous_aggregate(
```

Reply:
But it cannot be used in queries and, since it takes only single continuous aggregate and single chunk, it will be cumbersome to use. I suggest to improve the name of the function instead, but after this PR, before moving the function to public space.

Reply:
Makes sense. With the info below I think we should keep this as a function.

Reply:
@k-rus I am confused by this comment. If this is a function, it can be used in queries. 
Could we modify the name now instead of deferring? We now have 1 function and a procedure with the same name and it is not clear how they differ in functionality. We would certainly document this with an example in the docs. Having a better name would really help.

Reply:
@gayyappan There is no better proposed name. I made another name in the proposal, which I shared on Slack in database channel last week. Mine name there is not good either.

Since the function is internal, we don't need to deliver good name immediately.

Reply:
My suggestion for a name is refresh_continuous_aggregate_for_raw_chunk_boundary. It is a mouthful but I think the semantics is clear. Since it is internal , we could shorten this to refresh_cagg_for_raw_chunk.

Reply:
FWIW, IMO this function does the exact same thing as the other `refresh_continuous_aggregate`: namely it refreshes the cagg in the region specified. In one case the region is specified by an explicit window (start, end), in the second case the region is implicit by the chunk (still a start, end).

To only other difference is that the function in this PR can be used in a transaction, while the other procedure cannot (currently). But there is no reason why we cannot make the window-version transactional if it is executed within a transaction block.